tool/get-dart/dart_sdk_archive/lib/src/version_selector.dart

```diff
@@ -9,7 +9,7 @@ import 'package:sdk_builds/sdk_builds.dart';
 
 import 'operating_system.dart';
 
-const _storageBase = 'https://storage.googleapis.com/dart-archive';
+const _storageBase = '${storageBaseURL}dart-archive';
```

Comment:
Fix this to match your declaration:

```suggestion
const _storageBase = '${storageBaseUrl}dart-archive';
```